openshift/build.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
+kind: Template
+metadata: {}
+objects:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: tomcat9
```

Comment:
Introduce `NAME` and `SUFFIX` parameters and use them in .metadata.name and related fields.